client/my-sites/checkout/concierge-session-nudge/index.jsx

```diff
@@ -23,6 +24,10 @@ import isEligibleForDotcomChecklist from 'state/selectors/is-eligible-for-dotcom
 import { getSiteSlug } from 'state/sites/selectors';
 import { recordTracksEvent } from 'state/analytics/actions';
 
+/*
+ * There are no translate() calls in this file because it's launching only to
+ * EN audience
```

Comment:
We will actually want to translate this page. It'll first be tested on an en audience but will likely launch to non-en users soon after.